### PR TITLE
Box changes

### LINKED ISSUE #6559: [Trigger Warning] [Autism]
aaah

MY AUTISM, IT BURNS

### LINKED ISSUE #6681: Boxstation mail outlets
All the mail that gets sent to Security comes out of the disposal on the far end of the lounge, that ends up being right next to HoS mail outlet. The problem is that nobody ever goes back there and I'm pretty sure it's outside of audible range from where sec personnel in the brig usually hang out. I think it would make much more sense if mail instead came out of the other disposal outlet close to the entrance, where the warden can see t from his office and it can be heard by everybody in the brig hall.
image
The medbay mail outlet is also pretty far back, but all the other departments have their mail outlets in high traffic areas where everybody can see or hear them.

### PR DESCRIPTION
**Pull request by Probe1**
**Please see the comments for images.**
### Overhauls
- Significant alteration to botany back room and the adjacent maint to accommodate it. It was expanded by 1x9 tiles, the grinder was moved into the back room, and a paperbin, pen, labeler, folder, chair, and 3 tables were added.  The sink was removed to stop farm bots from pathing towards it and getting stuck in a loop. This has been requested for 6 months. Someone requested it in the thread and reminded me.  https://github.com/d3athrow/vgstation13/pull/6792#issuecomment-160723876
- Overhauls Research Botany at the Research Outpost.  The room is significantly changed in layout.  The two xenobotany machines are added.  A smartfridge, desk, and office supplies are added.  Connectors and a vent to space are added so the trays can remove dangerous gasses.  A hatchet and wrench are added to the plant crate.  Two plant analyzers are added.  https://github.com/d3athrow/vgstation13/pull/6792#issuecomment-160810513
### Tweaks
- Resolves my own issue https://github.com/d3athrow/vgstation13/issues/6542 by adding a new area, and a new icon for the area, and a new APC for the area known as permabrig.  I named it Permanent Confinement to keep in line with the lingo.  This will drastically reduce the power drain on the prison wing APC and bring the APCs in the brig in line with the rest of the station in terms of power drain.  I placed the APC on top of an existing power line because it was the closest free place available, allowing security to see it without entering perma.  The new area is immune to radiation storms, as it was before with the old area.  https://github.com/d3athrow/vgstation13/pull/6792#issuecomment-160544873
- I resolved https://github.com/d3athrow/vgstation13/issues/6681 by a slight modification of the existing pipe line and removing the other sort junction, supplanting it with a new one. https://github.com/d3athrow/vgstation13/pull/6792#issuecomment-160543498
- Fixes https://github.com/d3athrow/vgstation13/issues/6559 
- Adds a mailman outfit to an undisclosed location for players to find.  One guy can arrpee as a hounddog mail worker named Jjenjr.
- Adds two pairs of magboots to the research outposts Expedition Preparation Room.  
- Removes a random errant camera in maint above botany backroom that was labeled as a solar panel camera.  Someones unnoticed whoops.  This is why you don't have your cursor set to place mode when you're not placing things!
- Changes a reinforced table in the bar next to the booze-o-mat to a regular table so redecoration isn't as laborious.  Request came from the thread.

### CODE REVIEW

**Comment by Probe1** [comment]
I was merely pretending to be retarded in my first commit!  This should resolve it.

image

**Comment by Probe1** [comment]
new area, new icon for perma.
image

**Comment by Probe1** [comment]
Mailman outfit for anon in the thread.  Seemed like a nice idea.

**Comment by Probe1** [comment]
image

Significant alteration to botany back room and the adjacent maint to accommodate it. It was expanded by 1x9 tiles, the grinder was moved into the back room, and a paperbin, pen, folder, chair, and 3 tables were added. This has been requested for 6 months. Someone requested it in the thread and reminded me.

@9600bauds @TermedClepe please look over it and see what you think

**Comment by Probe1** [comment]
Added the new sec_perma area to the exception list for rad storms to keep those poor prisoners from dying.  Thank you for the reminder @ArthurDentist and the help on the other thing.  As far as I know it should be done now and I'm just waiting on feedback for botany.

**Comment by ghost** [comment]
Remove the sink from the botany back room. It makes botany bots stuck on the airlock trying to refill water.

Other than that looks good, everything in place.

**Comment by ihadtoregisterforthis** [comment]
It would look nicer if the APC wasn't inside the prison area. Perhaps moving it between the double airlocks?

**Comment by Probe1** [comment]
@ihadtoregisterforthis It's gotta be its just gotta!  No really it's a balance thing. Forcing a player to enter to use its air alarm and APC and possibly run into a prisoner creates interesting situations, more than not.

**Comment by 9600bauds** [comment]
> botany backroom

!!!YES!!!

Two things, however, I'm no atmosman but I think you might want to move the vent out from under the table. Second being switch the lysis-isolator and bioballistic delivery system around becuase it triggers my autism pls ok ty.
Otherwise, fantastic, absolutely fantastic

**Comment by Probe1** [comment]
ravioli ravioli give me the formuoli

Removes the sink from botany, switches the lysis and bioballistic machines, and modifies the waste line and scrubber to not be so spaghetti.

image

**Comment by 9600bauds** [comment]
never forgetti

**Comment by Probe1** [comment]
I don't want to admit it but I rather autistically positioned that labeler and paper bin so it looked nice rather than the usual shit heap of items we find on tables.  Unless @Kurfursten or anyone sees anything wrong I think the PR is complete.

**Comment by Kurfursten** [comment]
Pipes look good. I would move the table and chemmaster a square right. That frees up more space near the door, which is useful when you're moving exotic crates into the back room - especially when you have several of them.

**Comment by Probe1** [comment]
Works for me.  I put it that way because I believe having that hole in the tables made the room feel more open than it was, giving players the illusion of space even if they wouldn't use it.  But we can nudge them.

I added magboots to the research outposts expedition room which has been requested forever and universally wanted.  

image

**Comment by 9600bauds** [comment]
If you're moving the chemmaster, flip it and the grinder around for muh table flush

**Comment by Probe1** [comment]
That's what I did yes.

**Comment by Probe1** [comment]
I significantly altered xenobotany at the research outpost to accommodate the xenobiology machines and make it a functional addition instead of a cupboard people go to snack on wild mushrooms and ignore.  Picture after compile.

**Comment by Probe1** [comment]
Research Botany/ Xenobot.  The >paw in the top screenshot is gone, the bottom dreammaker image is current.

west_object_48_2015-11-30_19-43-48

vgstation13_ mapstgstation dmm _-_dream_maker_2015-11-30_22-18-00

**Comment by Kurfursten** [comment]
Why is there a furry paw in tiles? Designated yiff zone?

**Comment by Probe1** [comment]
>furry paws
well that's ruined for me now.  The map tools are missing the correct tile sprites.  I made an arrow.  Or  paws.  It adds texture to the room.

**Comment by Kurfursten** [comment]
It's not missing the correct sprite, talk to me on irc

**Comment by ghost** [comment]
I'm legitimately mad at science getting xenobotany with chem dispenser next to it.

Why would I want to play a botanist anymore when I can do the exact same thing just better at science?

**Comment by Kurfursten** [comment]
@TermedClepe He was talked down from giving them 16 trays, so at least botany still has
- Easier mail from Cargo for exotics and other stuff
- 2.5x more trays

**Comment by ghost** [comment]
@Kurfursten
Tray count is a joke since you can have infinite trays from grass.

Science botany has a fucking chem dispenser right in it.

**Comment by Probe1** [comment]
@TermedClepe We've had a research botanist job for a long time.  They got a dogshit cupboard without xenobotany machines.  Their job was pointless.

I don't like the job.  I think it shouldn't exist.  But it does and people have been begging for the room to be re done.  If you want it removed, make a discussion issue, get some people talking, I'll be right there supporting it.  But as long as it's still in the game then it needs to have parity with the botanist.

**Comment by Kurfursten** [comment]
I guess. But by the same token, radium is really easy to get as a normal botanist.

In Botany2, a chem dispenser isn't really a big deal since it's all just dice rolls to get stat ups anyway, but it will be **_huge**_ when Botany3 hits.

@Probe1 Might be worth talking about killing Research Botanist in the Botany3 PR. Why don't you do some polling to see how much people even want to keep it?

**Comment by Probe1** [comment]
More people wanted to keep it than remove it which is why I just spent 3 hours mapping.  My precise experience is until @TermedClepe just started posting in the thread and this PR today, I was the only person I knew of that did not like research botanists.  The only time it is ever discussed is players complaining about how badly the research outposts botany is implemented.

**Comment by Probe1** [comment]
And to fight for my own changes:  The only major additions here are the xenobotany machines (the _point_ of being a research botanist is to use them), two free exotic seeds, two additional trays (which is still far behind Baystation who have 16) and a lot better layout of the room.

**Comment by ghost** [comment]
> free exotic seeds

I can't be more mad.

**Comment by Probe1** [comment]
I will remove them if it's a sticking point.  I didn't think 2 was egregious when a crate comes with 6-8 and often times ends up with no good exotic seeds.

**Comment by ghost** [comment]
It's ok you can leave it.

The power creep will become clearer once botany overhaul gets merged. I'll make a new issue after that as this is basically a derailment about the bigger problem that is research botany.

**Comment by Probe1** [comment]
Too late they're gooooooone.  Everything is open to feedback.

**Comment by ghost** [comment]
I think science could really use an increased job cap at round start since it has so many different things to do.

Probably couldn't even fill all the roles with current cap.

Man I feel like shit just pissing over your mapping efforts now, sorry about that.

**Comment by Kurfursten** [comment]
Science could do with some role-splitting to be perfectly honest. I've seen a shift where THREE Xenoarcheologists joined. Best of all? The mechanic decided to abandon his workshop to play xenoarch. Meaning two xenoarchs had no access to a suit to do their job.

**Comment by Probe1** [comment]
I'm sure they were comfortable in their NASA Voidsuits.

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -2079,14 +2083,14 @@ proc/process_adminbus_teleport_locs()
 	if(ismob(Obj))
 		if(Obj:client)
 			mysound.status = SOUND_UPDATE
-			Obj << mysound
+			Obj << mysound
```
Comment (Probe1) [inline code/game/area/Space Station 13 areas.dm:2086]: What are all these?  Did the map cleaner decide to fuck with me?

File: html/changelogs/example.yml [hunk 2]
```diff
@@ -12,8 +12,8 @@
 #   tweak
 #   soundadd
 #   sounddel
-#   rscdel (general adding of nice things)
-#   rscadd (general deleting of nice things)
+#   rscdel (general deleting of nice things)
+#   rscadd (general adding of nice things)
```
Comment (Intigracy) [inline html/changelogs/example.yml:16]: fag, leave our memes alone

Comment (Probe1) [inline html/changelogs/example.yml:16]: no i must fix things

**Comment by Probe1** [comment]
The dumb paw design in research bot was removed by the way, didn't explicitly state it.  dreammaker screenshot is updated to reflect it.

**Comment by 9600bauds** [comment]
What the christ is this talk about a designated yiffing zone

**Comment by Probe1** [comment]
we should rename the TEG burn chamber as the yiff zone.

**Comment by 9600bauds** [comment]
No really what the dickens did I miss

**Comment by Probe1** [comment]
Nothing.   `dir` confused me and I mistakenly thought there wasn't the correct tile paint option.  So I did a quick little arrow design.  Kurfurst pointed out it looked like a paw, then pointed out that `dir` went up to 10 rather than 8 as I believed.  I corrected the tile paint that was wrong and removed the pawrrow.

**Comment by Probe1** [comment]
Two minor fixes.

Fixes an errant research hall area in the research shuttle bay, causing fires to trigger alarm overlays in a safe area.  Also removes the weird ass tiles in the bar that merely LOOKED like wooden tiles, but were really metal tiles painted to appear as wood.  I don't even know man.

**Comment by Probe1** [comment]
ok i'm done no more.  I need to move on to a new map.

File: code/game/area/Space Station 13 areas.dm (hunk 1 above)
Comment (PJB3005) [inline code/game/area/Space Station 13 areas.dm:2086]: Hrm.

It might be as a result of the `to_chat()` refactor in some way?

**Comment by Probe1** [comment]
Alright 24 hours have passed and the only substantive feedback I've received from posting it in OOC/thread is that @TermedClepe doesn't want Research Botanists to be a thing, which is a separate issue.  And @SirCrab reminding me to add detective manuals to all maps (and fix the code for them) _after_ his PR to fix the console is merged, which is for a separate PR after his.

**Comment by 9600bauds** [comment]
merge now before confrictu

**Pull request merged**